app/appmessage/rpc_get_balance_by_address.go

```diff
@@ -30,7 +30,7 @@ type GetBalanceByAddressResponseMessage struct {
 
 // Command returns the protocol command string for the message
 func (msg *GetBalanceByAddressResponseMessage) Command() MessageCommand {
-	return CmdGetBalanceByAddressResponseMessage
+	return CmdGetBalancesByAddressesResponseMessage
```

Comment:
same here, seems like a mistake and it should remain `CmdGetBalanceByAddressResponseMessage`